misc/helper-scripts/taste-create-data-view

```diff
@@ -97,7 +117,60 @@ fi
 
 # cp $(taste-config --prefix)/share/taste-types/taste-types.asn .
 
-taste-update-data-view
+# If ROS messages will be generated
+if [ $GEN_ROS_MSGS ]
+then
+    # Generate TASTE extended types
+    echo 'TASTE-ExtendedTypes DEFINITIONS ::=
+    BEGIN
+
+    IMPORTS
+    T-Int32, T-UInt32, T-Int8, T-UInt8, T-Boolean  FROM TASTE-BasicTypes;
+
+    -- Set of TASTE predefined basic types
+
+    T-Double ::= REAL (-1.7976931348623157e+308..1.7976931348623157E308)
+
+    T-Float ::= REAL (-3.402823466E+38..3.402823466e+38)
+
+    T-Int16 ::= INTEGER (-32768..32767 )
+
+    T-UInt16 ::= INTEGER (0 .. 65535 )
+
+    T-UInt64 ::= INTEGER (0..9223372036854775807)
```

Comment:
The range seems insufficient for this type. `(0..18446744073709551615)`

Reply:
Correct. That was my impression as well but the ASN.1 compiler complains with the correct range...

Reply:
Strange. Does it accept 9223372036854775808 ? If it does, it gets very weird, otherwise it would make sense to assume that the ASN.1 compiler internally uses int64_t to represent integer literals rather than some arbitrary-precision number format.

Reply:
Nop doesent. Just tested it.